src/py42/clients/detectionlists/detection_list_user_client.py

```diff
@@ -0,0 +1,180 @@
+import json
+
+from py42._internal.compat import str
+from py42.clients import BaseClient
+
+
+class DetectionListUserClient(BaseClient):
+    """Administrator utility to manage High Risk employees information.
+
+    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
+    """
+
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+    _resource = u"/user"
+
+    def __init__(self, session, user_context):
+        super(DetectionListUserClient, self).__init__(session)
+        self._user_context = user_context
+
+    def _make_uri(self, action):
+        return u"{0}{1}{2}".format(self._uri_prefix, self._resource, action)
+
+    def create(self, username):
+        """Create a detection list profile for a user.
+
+        Args:
+            username (str): Username of the user.
+
+        Returns:
+            :class:`py42.sdk.response.Py42Response`
+        """
+        data = {
+            "tenantId": self._user_context.get_current_tenant_id(),
```

Comment:
Do we want to still allow users to specify a tenant ID if they want?
@alanag13

Reply:
An example would be departing employee client - it only gets the current tenant ID if none is passed in.